src/libcrun/error.c

```diff
@@ -236,7 +236,7 @@ libcrun_init_logging (crun_output_handler *new_output_handler, void **new_output
 
         case LOG_TYPE_JOURNALD:
           *new_output_handler = log_write_to_journald;
-          *new_output_handler_arg = NULL;
+          *new_output_handler_arg = (void *) id;
```

Comment:
An idea...

Looking at the callback signature:

```c
void log_write_to_journald (int errno_, const char *msg, int verbosity, void *arg arg_unused)
```

We could rename the `arg_unused` to something else now, especially since it will be used.

Reply:
Makes sense, I renamed it to `id`.

Reply:
Ah I had to revert since it's still technically unused when building without systemd support.